src/porepy/numerics/fem/rt0.py

```diff
@@ -53,7 +53,7 @@ def discretize(self, sd: pp.Grid, data: dict) -> None:
             matrix_dictionary[self.div_matrix_key] = sps.csr_matrix(
                 (sd.num_faces, sd.num_cells)
             )
-            matrix_dictionary[self.vector_proj_key] = sps.csr_matrix((3, sd.num_cells))
+            matrix_dictionary[self.vector_proj_key] = sps.csr_matrix((3, 0))
```

Comment:
To be precise, is it 0 or `sd.num_faces` (which is zero)?

Reply:
Since in higher dimensions the vector_proj reconstructs from the dofs on the faces P0 vectors for each cells, I would say it's actually 0. It's really specific for the 0d mesh.

Reply:
*but there are no faces in 0d so no dofs available to reconstruct any velocity